Example/Pods/PromiseKit/Sources/after.swift

```diff
@@ -0,0 +1,12 @@
+import struct Foundation.TimeInterval
+import Dispatch
+
+/**
+ - Returns: A new promise that fulfills after the specified duration.
+*/
+public func after(interval: TimeInterval) -> Promise<Void> {
```

Comment:
Should this maybe have been:

    public func after(seconds: TimeInterval) -> Promise<Void>

and

    public func after(interval: DispatchTimeInterval) -> Promise<Void>